src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1599,11 +1598,15 @@ def client_side_proxy_wrapper(cmd,
                               context_name=None,
                               api_server_port=consts.API_SERVER_PORT):
 
+    cloud = send_cloud_telemetry(cmd)
+    if cloud == consts.Azure_USGovCloudName:
+        telemetry.set_debug_info('User tried proxy command in fairfax')
+        raise ClientRequestError(f'Cluster Connect feature is not yet available in {consts.Azure_USGovCloudName}')
```

Comment:
if we are raising an exception, we should collect exception telemetry as well. Define a fault type for this error

Reply:
added a telemetry exception as you asked.